extensions/wikia/Search/WikiaSearchAjaxController.class.php

```diff
@@ -27,9 +27,7 @@ public function getNextResults() {
 		$crossWikia = $this->request->getBool( 'crossWikia' );
 		$hub = false;
 
-		$isCorporateWiki = !empty( $this->wg->EnableWikiaHomePageExt );
-
-		$isInterWiki = $crossWikia ? true : $isCorporateWiki;
+		$isInterWiki = $crossWikia ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)